src/hooks/useDebounce.ts

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+
+export default <T>(value: T, ms: number) => {
```

Comment:
I actually implemented both versions, and the second time I figured it made more sense for the hook to debounce a value rather than a callback's result.
As per your suggestion I have now installed [usehooks](https://github.com/juliencrn/usehooks-ts) instead of having our own implementation.